person.rb

```diff
@@ -17,6 +20,10 @@ def can_use_services?
     end
   end
 
+  def correct_name
+    @name
+  end
```

Comment:
- Hey, in here, the person is inheriting from the Nameable, you should call the method here `@name.correct_name`. Could you improve on this?